src/status_im/chat/models/link_preview.cljs

```diff
@@ -0,0 +1,46 @@
+(ns status-im.chat.models.link-preview
+  (:require [re-frame.core :as re-frame]
+            [status-im.utils.fx :as fx]
+            [status-im.multiaccounts.update.core :as multiaccounts.update]
+            [status-im.native-module.core :as native-module]))
+
+(fx/defn set-link-preview
+  [{{:keys [multiaccount]} :db :as cofx} site enabled?]
+  (fx/merge cofx
+            (multiaccounts.update/multiaccount-update
+             :link-previews-enabled-sites
+             (if enabled?
+               (conj (get multiaccount :link-previews-enabled-sites #{}) site)
+               (disj (get multiaccount :link-previews-enabled-sites #{}) site))
+             {})))
+
+(fx/defn cache-link-preview-data
```

Comment:
could you pls elaborate on this ?

Reply:
During session we save locally data about link preview to not fetch it every time when chat re-creates